### ISSUE TITLE
the application downloads the first 99 songs from the playlist

### ISSUE THREAD

**Issue by Nokoster** [sheensantoscapadngan/SpotaFree#8]
is there any chance to increase this limit in the future?

**Comment by GitStudying** [comment]
It is possible, it has to do with the request, for more information see: https://stackoverflow.com/questions/39086287/spotipy-how-to-read-more-than-100-tracks-from-a-playlist
To go over this limit, the best solution seems to use offset of 100 and do this for every 100 tracks. I couldn't exactly find the code where this happens so I can't propose a direct solution as pull request.

**Comment by supersu-man** [comment]
Yep, I've removed the limit in the code, you can check [this](https://github.com/supersu-man/SpotaFree) out.
If you don't want to download the Video music version of music but the Audio only version then check [this](https://github.com/supersu-man/SpotifyPlaylistDownloader) out.
